proxy/http/remap/unit-tests/combined_no_weight_alias_extension.yaml

```diff
@@ -0,0 +1,76 @@
+#/** @file
+#
+#  A brief file description
+#
+#  @section license License
+#
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# */
+#
+# combined_no_weight_alias_extension.yaml
+#
+hosts:
+  p1: &p1
+    host: p1-cache.foo.com
+    protocol:
+      - http: 80
+      - https: 443
+    healthcheck:
+      url: tcp://192.168.1.1:80
+  p2: &p2
+    host: p2-cache.foo.com
+    protocol:
+      - http: 8080
+      - https: 8443
+    healthcheck:
+      url: tcp://192.168.1.2:80
+  s1: &s1
+    host: s1-cache.bar.com
+    protocol:
+      - http: 80
+      - https: 443
+    healthcheck:
+      url: tcp://192.168.2.1:80
+  s2: &s2
+    host: s2-cache.bar.com
+    protocol:
+      - http: 8080
+      - https: 8443
+    healthcheck:
+      url: tcp://192.168.2.2:80
+groups:
+  - g1: &g1
+    - *p1
+    - *p2
+  - g2: &g2
+    - *s1
+    - *s2
+strategy:
+  policy: consistent_hash
+  hash_key: path+query
+  groups:
+    - *g1
+    - *g2
+  protocol: http
+  failover:
+    ring_mode: exhaust_rings
+    response_codes:
+      - 404
+      - 503
+    health_check:
```

Comment:
Why is this a list? Can you have multiple health-check methods?

Reply:
I know many of the commercial DNS load balancers allow for multiple ways of health checking.

Reply:
yes, you can specify, 'active' and 'passive'.